src/components/withProps.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+
+const withProps = (WrappedComponent, outerProps) => {
+  class LatticeWrapper extends Component {
+    render() {
+      return (
+        <WrappedComponent
+          {...outerProps}
+          {...this.props} />
```

Comment:
What happens if you have a duplicate prop between `outerProps` and `this.props`?

Reply:
this.props should override the props supplied by the HOC. I will adjust the naming of outerProps to be less confusing and add a test for this.